typescript/demo/demo.ts

```diff
@@ -70,20 +70,18 @@ async function openAIWithAIConfig() {
   //     console.log(aiConfig.getOutputText("demoPrompt", output));
   //   }
 
+  const streamCallback = (data: any, _accumulatedData: any, _index: any) => {
```

Comment:
Minor cleanup:
- use defined `streamCallback` here, since it was just an unused variable before
- remove unnecessary `let result2` below

Technically should be a separate diff but I had to test this script out and easier to just include these small things here